media/admin/com_virtuemart/views/zasilkovna/view.html.php

```diff
@@ -248,4 +251,29 @@ public function renderShipmentsList()
 
         return VmHTML::select('order_exported', $objList, $selected_shipment, 'class="inputbox" onchange="resetTaskAndSubmitForm(this.form);"');
     }
+
+    /**
+     * @return void
+     */
+    private function showDeprecationWarning()
+    {
+        $app = JFactory::getApplication();
+        $readmeDeprecationLink['en'] = 'https://github.com/Zasilkovna/virtuemart3/blob/master/README.md#warning---feature-drop-plan-delivery-and-payment-limitations-settings';
+        $readmeDeprecationLink['cz'] = 'https://github.com/Zasilkovna/virtuemart3/blob/master/README.md#upozorn%C4%9Bn%C3%AD---pl%C3%A1n-odstran%C4%9Bn%C3%AD-funkce-omezen%C3%AD-dopravy-a-platby';
+
+        $langTag = JFactory::getLanguage()->getTag();
```

Comment:
Může se stát, že JFactory::getLanguage() vrátí null? Pokud ne, udělal bych z toho proměnnou opatřenou phpdocem.
To samé u JFactory::getApplication().